content/blog/2021-03-03-dvc-2-0-release.md

```diff
@@ -2,11 +2,11 @@
 title: DVC 2.0 Release
 date: 2021-03-03
 description: |
-  Today is DVC 2.0 release day! Watch a video from DVC-team when we explain the 
+  Today is DVC 2.0 release day! Watch a video from DVC-team when we explain the
   new features and read more details in this blog post.
 
 descriptionLong: |
-  Today is DVC 2.0 release day! Watch a video from DVC-team when we explain the 
+  Today is DVC 2.0 release day! Watch a video from DVC-team when we explain the
```

Comment:
btw my text editor automatically removes these. Why didn't this repo's linters complain earlier though? CC @jorgeorpinel / @shcheklein I thought this was fixed?

Reply:
Idk if it was in place in March